styles/x-terminal.less

```diff
@@ -106,16 +114,17 @@ x-terminal-profile {
 	max-width: 100%;
 	margin-bottom: 5px;
 	background-color: @profile-menu-background-color;
+	margin-left: 2px;
 }
 
 .x-terminal-profile-menu-item-label-checkbox {
-	padding-left: 2.25em;
+	padding-left: 2.2em;
 	margin-top: -20px;
 
-	.x-terminal-profile-menu-item-checkbox {
-		width: 15px;
-		height: 15px;
-		margin-left: -2.25em;
+	.input-checkbox {
```

Comment:
It only works here like this. Anything else breaks.

Reply:
Why would it not work the same if we add the classname `x-terminal-profile-menu-item-checkbox` and style that class instead of `input-checkbox`?

If we are running into precedence issues you can use `!important` to override the styling from `input-checkbox`.

> adding x-terminal to sub classes is useless.

It is future-proofing incase someone decides they shouldn't be nested and breaks other elements with the same class names.

Reply:
> If we are running into precedence issues you can use `!important` to override the styling from `input-checkbox`.

And you think I didnt try that?

> It is future-proofing incase someone decides they shouldn't be nested and breaks other elements with the same class names. Or some other package decides to give their element the same class and not nest their styles.

We can account for many things in the design, I never heard of such issues personally with Atom addons and forks, The way things are loaded with unique paths it would be insane of Atom to suddenly wack everything in a design that would cause these things to happen.

If it would happen, it would be an Electron/Atom issue. Besides it can be easily undone or redone its just CSS.

Reply:
Do the check boxes look different with the fixes I pushed?

> Besides it can be easily undone or redone its just CSS.

Yes but it will take some time to figure out where the issue is and why it is happening. It doesn't hurt anything to have classes prefixed with`x-terminal`

Reply:
I still think the lot as is is overkill for the sake of unlikely situations.